.github/workflows/api-php-static-code-check.yml

```diff
@@ -7,18 +7,9 @@ jobs:
     continue-on-error: true
     strategy:
       matrix:
-        php-versions: [ '7.4', '8.0' ]
-        nextcloud: [ 'stable24' ]
+        php-versions: [ '7.4', '8.0', '8.1' ]
+        nextcloud: [ 'stable25' ]
         database: [ 'sqlite' ]
-        include:
-          - php-versions: 8.1
```

Comment:
Let's do 8.2 with this now

Reply:
Yes good idea, will do that later

Reply:
PHP 8.2 would be required?

Reply:
No that is only for the automated tests 8.2 will not be required

Reply:
As soon as the beta is finished with those automatic check I will download it and test it.